aiohttp_asgi_connector/transport.py

```diff
@@ -127,6 +127,14 @@ async def send(message: "MutableMapping[str, Any]") -> None:
     def _encode_response(
         self, status: int, headers: "List[Tuple[bytes, bytes]]", body: bytearray
     ) -> bytes:
+        # add content-length if not present, which happens with streamed responses
+        found = False
+        for (name, value) in headers:
+            if name.lower() == b"content-length":
+                found = True
+        if not found:
+            headers.append((b"content-length", str(len(body)).encode()))
+
```

Comment:
would you mind pointing me to the relevant part of the asgi spec or some reference implementation where this is the correct method to deal with this problem?